### ISSUE TITLE
mineable pickaxe tags list replacing instead of appending

### ISSUE THREAD

**Issue by WhiskeeDev** [SamsTheNerd/ducky-periphs#9]
In version [7.0.0 of my modpack](https://www.curseforge.com/minecraft/modpacks/tophat-modpack/files/all), ducky periphs version 1.1.4 replaces the [pickaxe mineables](https://github.com/SamsTheNerd/ducky-periphs/blob/main/src/main/resources/data/minecraft/tags/blocks/mineable/pickaxe.json) instead of appending to it - resulting in only ducky blocks being mineable with a pickaxe.

Removing the mod fixes the issue, (which is what hotfix 7.0.1 will be doing) and no other mod has this sort of behaviour.

My current theory is mod incompatability, but my pack has 182 mods so it's going to take some time for me to figure out what exactly (if I even can)... If I happen to get more information I'll append it onto this ticket.

**Comment by SamsTheNerd** [comment]
huh weird. Maybe try using REI or something to check if anything else is also in the tag ? (You can press `r` on any pickaxe mineable block, go over to tags and then expand to see everything) Maybe also check some of the other mineable tags maybe?

I don't think there's anything non standard about the pickaxe.json tag file and I can't reproduce the issue so there's not much I can do. Let me know if you find anything else though. 

Also if it helps, I haven't had any issues with using this in all of fabric 6, idk if that could help narrow down the problem.

**Comment by Ringlings** [comment]
Any chance that you could provide a startup log? It might show an error somewhere about it.

**Comment by MAGGen-hub** [comment]
Same problem... Can't mine anything...

**Comment by Ringlings** [comment]
> Same problem... Can't mine anything...

Are you using the pack above or a different one? Can you send a log file?

**Comment by WhiskeeDev** [comment]
> Any chance that you could provide a startup log? It might show an error somewhere about it.

ok sorry for taking so long to give this @Ringlings, ADHD sucks ass.

I read through the startup log and found the following which I think is our culprit for this issue:
```
[15:59:30] [Worker-Main-5/ERROR]: Couldn't load tag minecraft:mineable/pickaxe as it is missing following references: ducky-periphs:focal_port_block (from Ducky Peripherals)
```

Here is the whole latest.log file incase it's useful for additional info:
https://gist.github.com/WhiskeeDev/cf54f4c626540b66397b47964f555fff

**Comment by SamsTheNerd** [comment]
oh i see, focal port only gets registered when hex casting is installed. 

I'll look into the best way to fix that in the next release but if you want a quick fix you can install hex casting (with the bonus that you get another cool mod to play with lol)

**Comment by MAGGen-hub** [comment]
> oh i see, focal port only gets registered when hex casting is installed.
> 
> I'll look into the best way to fix that in the next release but if you want a quick fix you can install hex casting (with the bonus that you get another cool mod to play with lol)

Eh, sorry but... It's not a good solution for a create mod based modpack with 270+ mods.... I already have Eden Ring mod that can be used as infinite overpowered source of metal ores. If I install Hex Casting... My modpack may lose the last pieces of logic, balance, and stability.

I will try it but... Well... Hope I will not crash on the start...

**Comment by MAGGen-hub** [comment]
I think removing `ducky-periphs:focal_port_block` from `minecraft:mineable/pickaxe` may be better solution for persons who don't use Hex Casting... At last it worked for me...
```
{
    "replace": false,
    "values": [
        "ducky-periphs:weather_machine_block",
        "ducky-periphs:entity_detector_block",
        "ducky-periphs:sculkophone_block"
    ]
  }
```

**Comment by MAGGen-hub** [comment]
I am not very good at modding but... Well gues it imposible to make "dunamic" compatability tag for focal port block with a datapack... I saw something like `/data/yourmod/compatdata/compatmod/minecraft/tags/blocks/minable/pickaxe.json` but it was on 1.12.2 with some lib usage and on forge... 

I think you need to make it minable, by java code, unfortunately...

**Comment by SamsTheNerd** [comment]
yeah, either that or register a dummy focal port block if hexcasting isn't installed so it doesn't error out. I think the dummy block method would be better but idk, i'll look into it again after I do the architectury rework.

**Comment by SamsTheNerd** [comment]
should be fixed in v1.2.0

**Comment by dnk1234567** [comment]
This bug can be reproduce on 1.20.1 by minimal environment.
Mods list: 
duckyperiphs-1.20.1-1.3.1-forge
hexcasting-forge-1.20.1-0.11.1-7-pre-620
Forge 47.2.6

Installed the mod(Observer is just an example):
![2024-03-08_23 02 49](https://github.com/SamsTheNerd/ducky-periphs/assets/86105014/10400bb7-29eb-44d8-9065-6354c6372706)

Deleted the mod:
![2024-03-08_23 03 51](https://github.com/SamsTheNerd/ducky-periphs/assets/86105014/68315208-7c84-4ba2-8442-b843e1b30137)

The #minecraft:mineable/pickaxe is missing.
